bedrock/mozorg/templates/mozorg/advertising/principles.html

```diff
@@ -4,73 +4,54 @@
-        </li>
-        <li>
-          <h3>We treat your data like YOUR data</h3>
-          <p>We do not sell your personal data. And unlike big tech, we don’t use your age, gender, or precise location for ads.</p>
-          <p>We do need some information to make ads work, so we apply the same <a href="/privacy/principles/">data privacy principles</a> we use for the rest of Mozilla to ads. This means we’ll limit data collection, keep you in control of your data, and protect it with strong privacy tools like encryption and differential privacy.</p>
-        </li>
-        <li>
-          <h3>Don’t like it? Turn it off.</h3>
-          <p>You always have control over your ads experience. You can choose the data you’re okay to share or even opt-out of ads completely. And unlike big tech, our browser still supports the most powerful ad blockers available.</p>
-        </li>
-        <li>
-          <h3>See for yourself</h3>
-            <p>Firefox is open-source, so the community can verify that we’re keeping the data and privacy promises that we make.</p>
-        </li>
-      </ul>
-  </div>
-</section>
-
+  </header>
+
+  <section class="m24-c-content">
+    <div class="mza-half-grid">
+      <div class="two-column-intro">
+        <h2 class="mza-section-heading">Ads that work for you </h2>
+        <p class="mza-section-text-lg">The internet relies on advertising to fund the sites and apps people care about, but today’s system favors big tech—often at the expense of your privacy. We’re building a new approach to advertising that is a fair deal for everyone.</p>
+
+      </div>
+
+        <ul class="two-column-detail-list">
+          <li>
+            <h3 class="mza-section-subheading">No creepy ads</h3>
+            <p>We don't allow cross-site tracking in our ads—no third-party cookies or device identifiers are shared with advertisers—so our ads won't follow you around the internet. And if you use Firefox, we’ll keep others from tracking you, too.</p>
+          </li>
+          <li>
+            <h3 class="mza-section-subheading">No annoying ad formats</h3>
+            <p>We won’t serve ads that get in your way of using Firefox. All our ads are also clearly labeled, because we’re not trying to trick you.</p>
+          </li>
+          <li>
+            <h3 class="mza-section-subheading">We treat your data like YOUR data</h3>
+            <p>We do not sell your personal data. And unlike big tech, we don’t use your age, gender, or precise location for ads.</p>
+            <p>We do need some information to make ads work, so we apply the same <a href="/privacy/principles/">data privacy principles</a> we use for the rest of Mozilla to ads. This means we’ll limit data collection, keep you in control of your data, and protect it with strong privacy tools like encryption and differential privacy.</p>
```

Comment:
```suggestion
            <p>We do need some information to make ads work, so we apply the same <a href="{{ url('privacy.principles') }}">data privacy principles</a> we use for the rest of Mozilla to ads. This means we’ll limit data collection, keep you in control of your data, and protect it with strong privacy tools like encryption and differential privacy.</p>
```

Using the URL helper will speed up loading by inserting the current locale so it doesn't need to be looked up each time. (It will also help avoid broken links if pages move around.